### ISSUE TITLE
Runtime error relating to atari.Action class 

### ISSUE THREAD

**Issue by ugurkanates** [NVlabs/cule#20]
Tensor.__contains__ only supports Tensor or scalar, but you passed in a <class 'torchcule_atari.AtariAction'>

Just trying bunch of examples provided by NVIDIA but seems sadly  there is a bug ?

python 3.7.0
I installed apex from github and did python setup install ( pip version was having issues?)
pytorch 1.4.0
cuda version 10.2

**Comment by ifrosio** [comment]
If you want to move quick, can you try the docker image? That should work out of the box.

**Comment by ugurkanates** [comment]
@ifrosio  yes thats what I did. I may also update my fork and do a PR that works with all current version of libraries and Ubuntu 20.04 LTS.  This project is too good to left out die.

**Comment by ifrosio** [comment]
For docker instructions we posted:

FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04 as base
...
RUN pip install torch==1.2.0

It looks like you are using different CUDA / pytorch versions. Any particular reason for this or can you try downgrading to CUDA 10 / pytorch 1.2 / python 3.6?

**Comment by ugurkanates** [comment]
@ifrosio  Well there are two things ,
 1 - able to use latest versions of torch
 2- For CUDA I believe NVIDIA handles that part , I don't think you can even downgrade CUDA no ?
I mean I can handle torch with simple conda/ virtual env but yeah atleast CUDA part is I believe should be updated or choseable. My GPU isn't even up to date it's just one of the old models came with laptops 2GB VRAM model.
